install/ossm-installing-openshift-service-mesh.adoc

```diff
@@ -11,6 +11,8 @@ Installing {ocp-short-name} {SMProductShortName} consists of three main tasks: i
 [WARNING]
 ====
 Before migrating from {SMProduct} 2, make sure you are not running {SMProduct} 3 and {SMProduct} 2 in the same cluster, because it causes conflicts unless configured correctly. To migrate from {SMProduct} 2, see xref:../migrating/checklists/ossm-migrating-read-me.adoc#ossm-migrating-read-me[Migrating from {SMProduct} 2.6].
```

Comment:
```suggestion
Before installing {SMProduct} 3, make sure you are not running {SMProduct} 2 in the same cluster, because it causes conflicts unless configured correctly. To migrate from {SMProduct} 2, see xref:../migrating/checklists/ossm-migrating-read-me.adoc#ossm-migrating-read-me[Migrating from {SMProduct} 2.6].
```